### ISSUE TITLE
git branch -a broken

### ISSUE THREAD

**Issue by my-master** [scmbreeze/scm_breeze#279]
```
$ git branch -a
_scmb_git_branch_shortcuts:10: command not found: ruby
```
Probably related to #200 

`git branch -r` works as expected

**Comment by my-master** [comment]
@HaleTom You are right, it looks like there is no ruby:
```
$ type ruby
ruby not found
```
So I guess I need to install it. Probably, you need to mention in README.md that not only SCM Breeze works faster with Ruby, but Ruby is also required for some git commands. Thanks!

**Comment by ghthor** [comment]
Agreed. Will add the note, or figure out how to factor out the dependency.
